tests/test_templates.py

```diff
@@ -177,34 +180,3 @@ def test_default_parameters(working_dir: Path) -> None:
     response = run_init(working_dir, "test_default_parameters")
 
     assert response.returncode == 0, response.stdout
-
```

Comment:
Also need to remove the corresponding folders in tests_generated for the remove tests